dff/script/core/message.py

```diff
@@ -103,3 +103,7 @@ class Message(DataModel):
 
     def __repr__(self) -> str:
         return " ".join([f"{key}='{value}'" for key, value in self.dict(exclude_none=True).items()])
+
+
+class MultiMessage(Message):
+    hypotheses: Optional[List[Message]] = None
```

Comment:
Maybe make `confidences` a field of `MultiMessage`?
```suggestion
    hypotheses: Optional[List[Message]] = None
    confidences: Optional[List[Message]] = None
```